### PR TITLE
Add 346 release notes

### PR DESCRIPTION
**Pull request by martint**
Fixes https://github.com/prestosql/presto/issues/5678

### CODE REVIEW

**Review by ebyhr — COMMENTED** [review]
Left only minor comments.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# Release 345 (XX Nov 2020)
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-346.md:1]: 346

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+* Fix issue when query could return invalid results, if some column references were pruned out during query optimisation. ({issue}`5618`)
+
+## Cassandra Connector Changes
+
+* Improve performance of `INSERT` queries with batch statement. The batch size can be configured via `cassandra.batch-size`
+  configuration property. ({issue}`5047`)
+
+## Hive Connector Changes
+
+* Support coercion between `varchar` types of different lengths. ({issue}`5484`)
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-346.md:38]: This is about table/partition schema mismatch, but it's not obvious from the bullet's text.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+
+## Iceberg Connector Changes
+
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+    
+## Kudu Connector Changes
+
+* Improve performance of queries having only `LIMIT` clause. ({issue}`3691`)
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-346.md:51]: What does "only" mean here?

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+## Oracle Connector Changes
+
+* Add support for setting column comments. ({issue}`5399`)
+* Allow enabling remarks reporting via `oracle.remarks-reporting.enabled ` config property. ({issue}`5720`)
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-346.md:56]: Is it obvious that "remarks reporting" refers to "column comments"?

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+# Release 345 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add SQL syntax for `ALTER TABLE SET AUTHORIZATION` to change the table owner. ({issue}`5717`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
```
Comment (findepi) [inline presto-docs/src/main/sphinx/release/release-346.md:10]: You may want to include https://github.com/prestosql/presto/issues/5678#issuecomment-721616202 next to this.

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 2 above)
Comment (phd3) [reply presto-docs/src/main/sphinx/release/release-346.md:46]: @findepi any of the following read better?

```
* Support reading partitions with `varchar(x)` typed columns when the column evolves to `varchar(y)` type
```

```
* Fix table/partition schema mismatch failure when a `varchar` typed column changes width during schema evolution.

Comment (findepi) [reply presto-docs/src/main/sphinx/release/release-346.md:46]: To be both are better than the current wording.

Comment (martint) [reply presto-docs/src/main/sphinx/release/release-346.md:48]: Referring to "schema evolution" implies some kind of action (which may or may not be the reason for the mismatch). How about something like this? 

```
* Fix failure when the declared length of a `varchar(n)` column in the partition schema differs from the table schema.
```

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 3 above)
Comment (martint) [reply presto-docs/src/main/sphinx/release/release-346.md:64]: @Praveen2112 ?

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 4 above)
Comment (martint) [reply presto-docs/src/main/sphinx/release/release-346.md:73]: Not to me. I had no idea they were related.

@losipiuk ?

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 2 above)
Comment (phd3) [reply presto-docs/src/main/sphinx/release/release-346.md:48]: sounds good to me.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+## Hive Connector Changes
+
+* Support coercion between `varchar` types of different lengths. ({issue}`5484`)
+* Add support for `ALTER TABLE SET AUTHORIZATION` SQL syntax to change the table owner. ({issue}`5717`)
+* Improve query performance by adding support for dynamic bucket pruning. ({issue}`5634`)
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+* If multiple metastore URIs are defined via `hive.metastore.uri` prefer connecting to one which was seen operational most recently. 
+  The change was done to limit query failure rate if many of defined metastores are misbehaving. ({issue}`5795`)  
+* Fix Hive view access when catalog name is other than "hive". ({issue}`5785)
```
Comment (ebyhr) [inline presto-docs/src/main/sphinx/release/release-346.md:55]: Append ` after 5785

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+## SPI Changes
+
+* Fix propagating of connector session properties to `ConnectorNodePartitioningProvider`. ({issue}`5690`)
+
+## Other Connector Changes
+
+* Improve performance of queries with `DISTINCT` or `LIMIT`, or with `GROUP BY` and no aggregate functions and `LIMIT`,
+  when the computation can be pushed down to the underlying database for the PostgreSQL, MySQL, Oracle, Redshift and 
+  SQL Server connectors ({issue}`5522`)
```
Comment (phd3) [inline presto-docs/src/main/sphinx/release/release-346.md:88]: missing '.' at the end

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 3 above)
Comment (Praveen2112) [reply presto-docs/src/main/sphinx/release/release-346.md:64]: It doesn't work if we have any filters along with the table scan in the query.

**Review by losipiuk — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
+# Release 345 (XX Nov 2020)
+
+## General Changes
```
Comment (losipiuk) [inline presto-docs/src/main/sphinx/release/release-346.md:3]: I added https://github.com/prestosql/presto/issues/5678#issuecomment-724013433

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+* Improve performance of queries with uncorrelated ``IN`` clauses. ({issue}`5582`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
+* Fix incorrect handling of negative offsets for the `time with time zone` type. ({issue}`5696`)
+* Fix incorrect result when casting `time(p)` to `timestamp(p)` for precisions higher than 6. ({issue}`5736`)
+* Fix incorrect query results when comparing a `timestamp` column with a `timestamp with time zone` constant. ({issue}`5685`)
+* Fix improper table alias visibility for queries that select all fields. ({issue}`5660`)
+* Fix failure when query parameter appears in a lambda expression. ({issue}`5640`)
+* Fix failure for queries containing `DISTINCT *` and fully-qualified column names in the `ORDER BY` clause. ({issue}`5647`)
+* Fix planning failure for certain queries involving `INNER JOIN`, `GROUP BY` and correlated subqueries. ({issue}`5846`)
+* Fix issue when `EventListener` was sometimes not called if query was aborted in initial phase of execution. ({issue}`5815`)
```
Comment (martint) [inline presto-docs/src/main/sphinx/release/release-346.md:21]: @losipiuk, what's the user-visible effect? Also, this description is too low-level. Maybe something like "Fix recording of query completion event when query is aborted early".

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+* Allow enabling remarks reporting via `oracle.remarks-reporting.enabled` configuration property. ({issue}`5720`)
+
+## PostgreSQL Connector Changes
+
+* Improve performance of queries comparing a `timestamp` column with a `timestamp with time zone` constants 
+  for `timestamp with time zone` precision higher than 3. ({issue}`5543`)
+
+## SPI Changes
+
+* Fix propagating of connector session properties to `ConnectorNodePartitioningProvider`. ({issue}`5690`)
```
Comment (phd3) [inline presto-docs/src/main/sphinx/release/release-346.md:85]: optional suggestion: `propagating ` --> `propagation`?

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 3 above)
Comment (phd3) [reply presto-docs/src/main/sphinx/release/release-346.md:67]: we could say `Improve performance of simple projection queries with LIMIT clause`.

**Review by phd3 — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 9 above)
Comment (losipiuk) [reply presto-docs/src/main/sphinx/release/release-346.md:21]: I think your suggestion is nice. This is user visible as events are consumed by users code (via provided `EventListener` implementations)

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 3 above)
Comment (martint) [reply presto-docs/src/main/sphinx/release/release-346.md:67]: "projection query" is an unfamiliar term for users.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 11]
```diff
@@ -0,0 +1,93 @@
...
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
+* Add support for window frames based on `GROUPS`. ({issue}`5713`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add `ALTER TABLE SET AUTHORIZATION` syntax to allow to change the table owner. ({issue}`5717`)
+* Make `EXPLAIN` more readable for queries containing `timestamp` or `timestamp with time zone`  constants. ({issue}`5683`)
+* Improve performance for queries with inequality conditions. ({issue}`2674`)
+* Improve performance of queries with uncorrelated ``IN`` clauses. ({issue}`5582`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
```
Comment (dain) [inline presto-docs/src/main/sphinx/release/release-346.md:13]: I believe this is a Hive change

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 3 above)
Comment (Praveen2112) [reply presto-docs/src/main/sphinx/release/release-346.md:67]: That looks good.

**Review by kokosing — APPROVED** [review]

**Review by dain — APPROVED** [review]

**Review by ebyhr — APPROVED** [review]

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 12]
```diff
@@ -0,0 +1,93 @@
+# Release 346 (XX Nov 2020)
+
+## General Changes
+
+* Add support for `RANGE BETWEEN <value> PRECEDING AND <value> FOLLOWING` window frames. ({issue}`609`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:5]: Might be easier to read as

> `RANGE BETWEEN ... PRECEDING AND ... FOLLOWING`

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 13]
```diff
@@ -0,0 +1,93 @@
...
+* Add support for window frames based on `GROUPS`. ({issue}`5713`)
+* Add support for {func}`extract` with `TIMEZONE_HOUR` and `TIMEZONE_MINUTE` for `time with time zone` values. ({issue}`5668`)
+* Add SQL syntax for `GRANT` and `REVOKE` on schema. This is not yet used by any connector. ({issue}`4396`)
+* Add `ALTER TABLE SET AUTHORIZATION` syntax to allow to change the table owner. ({issue}`5717`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:9]: > `ALTER TABLE ... SET AUTHORIZATION`

Comment (electrum) [reply presto-docs/src/main/sphinx/release/release-346.md:9]: "to allow changing the"

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 14]
```diff
@@ -0,0 +1,93 @@
...
+* Make `EXPLAIN` more readable for queries containing `timestamp` or `timestamp with time zone`  constants. ({issue}`5683`)
+* Improve performance for queries with inequality conditions. ({issue}`2674`)
+* Improve performance of queries with uncorrelated ``IN`` clauses. ({issue}`5582`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:12]: Single backticks

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 15]
```diff
@@ -0,0 +1,93 @@
...
+* Fix planning failure for certain queries involving `INNER JOIN`, `GROUP BY` and correlated subqueries. ({issue}`5846`)
+* Fix recording of query completion event when query is aborted early. ({issue}`5815`)
+
+## Web UI Changes
+
+* Fix "Capture Snapshot" button on the Worker page. ({issue}`5759`)
+
+## JDBC Driver Changes
+
+* Support number accessor methods like `ResultSet.getLong` or `ResultSet.getDouble`
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:38]: `ResultSet.getLong()`
`ResultSet.getDouble()`

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 16]
```diff
@@ -0,0 +1,93 @@
...
+  on `decimal` values, as well as `char` or `varchar` that can be unambiguously interpreted as numbers. ({issue}`5509`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:39]: Add "values"

> `char` or `varchar` values that can ...

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 17]
```diff
@@ -0,0 +1,93 @@
...
+* Add `SSLVerification` JDBC connection parameter that allows to skip SSL verification on `SSLVerification=NONE` or to skip SSL
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:40]: We can shorten this to
```
* Add `SSLVerification` JDBC connection parameter that allows configuring SSL verification.
```
The full docs are available on the JDBC page.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 18]
```diff
@@ -0,0 +1,93 @@
...
+* Support number accessor methods like `ResultSet.getLong` or `ResultSet.getDouble`
+  on `decimal` values, as well as `char` or `varchar` that can be unambiguously interpreted as numbers. ({issue}`5509`)
+* Add `SSLVerification` JDBC connection parameter that allows to skip SSL verification on `SSLVerification=NONE` or to skip SSL 
+  host name verification on `SSLVerification=CA`. ({issue}`5610`)
+* Remove legacy `useSessionTimeZone` JDBC connection parameter. ({issue}`4521`)
+* Implement `ResultSet.getRow()`. ({issue}`5769`)
+  
+## BigQuery Connector Changes
+
+* Fix issue when query could return invalid results, if some column references were pruned out during query optimization. ({issue}`5618`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:47]: Remove comma

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 19]
```diff
@@ -0,0 +1,93 @@
...
+
+## Cassandra Connector Changes
+
+* Improve performance of `INSERT` queries with batch statement. The batch size can be configured via `cassandra.batch-size`
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:51]: "via the"

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 20]
```diff
@@ -0,0 +1,93 @@
...
+  configuration property. ({issue}`5047`)
+
+## Elasticsearch Connector Changes
+
+* Fix failure when index mappings do not contain a `properties` section. ({issue}`5807`)
+
+## Hive Connector Changes
+
+* Add support for `ALTER TABLE SET AUTHORIZATION` SQL syntax to change the table owner. ({issue}`5717`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:60]: > `ALTER TABLE ... SET AUTHORIZATION`

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 21]
```diff
@@ -0,0 +1,93 @@
...
+* Improve query performance by adding support for dynamic bucket pruning. ({issue}`5634`)
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
+* If multiple metastore URIs are defined via `hive.metastore.uri` prefer connecting to one which was seen operational most recently.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:67]: Add comma after config property name

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 22]
```diff
@@ -0,0 +1,93 @@
...
+
+## Hive Connector Changes
+
+* Add support for `ALTER TABLE SET AUTHORIZATION` SQL syntax to change the table owner. ({issue}`5717`)
+* Improve query performance by adding support for dynamic bucket pruning. ({issue}`5634`)
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
+* If multiple metastore URIs are defined via `hive.metastore.uri` prefer connecting to one which was seen operational most recently. 
+  The change was done to limit query failure rate if many of defined metastores are misbehaving. ({issue}`5795`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:68]: > This prevents query failures when one or more metastores are misbehaving.

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 23]
```diff
@@ -0,0 +1,93 @@
...
+## Hive Connector Changes
+
+* Add support for `ALTER TABLE SET AUTHORIZATION` SQL syntax to change the table owner. ({issue}`5717`)
+* Improve query performance by adding support for dynamic bucket pruning. ({issue}`5634`)
+* Remove deprecated `parquet.fail-on-corrupted-statistics` (previously known as `hive.parquet.fail-on-corrupted-statistics`).
+  A new configuration property, `parquet.ignore-statistics`, can be used to deal with Parquet files with incorrect metadata.  ({issue}`3077`)
+* Do not write min/max statistics for `timestamp` columns. ({issue}`5858`)
+* If multiple metastore URIs are defined via `hive.metastore.uri` prefer connecting to one which was seen operational most recently. 
+  The change was done to limit query failure rate if many of defined metastores are misbehaving. ({issue}`5795`)  
+* Fix Hive view access when catalog name is other than "hive". ({issue}`5785`)
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:69]: Use code quotes for `hive`

File: presto-docs/src/main/sphinx/release/release-346.md [hunk 24]
```diff
@@ -0,0 +1,93 @@
...
+
+* Add support for setting column comments. ({issue}`5399`)
+* Allow enabling remarks reporting via `oracle.remarks-reporting.enabled` configuration property. ({issue}`5720`)
+
+## PostgreSQL Connector Changes
+
+* Improve performance of queries comparing a `timestamp` column with a `timestamp with time zone` constants 
+  for `timestamp with time zone` precision higher than 3. ({issue}`5543`)
+
+## SPI Changes
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-346.md:97]: Move this section last

File: presto-docs/src/main/sphinx/release/release-346.md (hunk 12 above)
Comment (martint) [reply presto-docs/src/main/sphinx/release/release-346.md:5]: I agree, but this is specifically about the case where a value is provided (vs what we support today: UNBOUNDED and CURRENT ROW)

**Comment by losipiuk** [comment]
Added https://github.com/prestosql/presto/issues/5678#issuecomment-724635858

**Comment by losipiuk** [comment]
Added https://github.com/prestosql/presto/issues/5678#issuecomment-724671666
